x-pack/plugins/observability_solution/slo/server/services/summary_transform_generator/helpers/create_temp_summary.ts

```diff
@@ -66,6 +67,17 @@ export function createTempSummaryDocument(
       name: apmParams?.transactionName ?? null,
       type: apmParams?.transactionType ?? null,
     },
+    // synthetics specific fields
+    monitor: {
+      name: null,
+      config_id: null,
+    },
+    observer: {
+      name: null,
+      geo: {
+        name: null,
+      },
+    },
```

Comment:
We should add them in the EsSummaryDoc type above like we did for APM's fields. This might break a few places (maybe some test fixtures)